src/containerapp/azext_containerapp/tests/latest/test_containerapp_env_commands.py

```diff
@@ -455,4 +455,41 @@ def test_containerapp_env_internal_only_e2e(self, resource_group):
         self.cmd(f'containerapp env show -n {env} -g {resource_group}', checks=[
             JMESPathCheck('name', env),
             JMESPathCheck('properties.vnetConfiguration.internal', True),
+        ])
+
+
+    @AllowLargeResponse(8192)
+    @ResourceGroupPreparer(location="northeurope")
+    @live_only() # passes live but hits CannotOverwriteExistingCassetteException when run from recording
```

Comment:
May I ask why do you need to mark this test as `@live_only()`?

Reply:
It hits CannotOverwriteExistingCassetteException when run from recording, just like all other existing tests in this file.

Reply:
Actually, there are many situations that may cause this `CannotOverwriteExistingCassetteException` error. Could you please tell me more specific reasons?

Reply:
TBH I do not know the exact reason as well. Locally it can pass without live_only but it fails on azure pipeline. Let me comment it out and rerun to show you,

Reply:
nvm, seems it's passing now...